org.eclipse.swt.win32.win32.x86/src/org/eclipse/swt/ole/win32/OleClientSite.d

```diff
@@ -465,7 +465,7 @@ protected void addObjectReferences() {
         objIOleLink.Release();
     }
 }
-protected int AddRef() {
+package int AddRef() {
```

Comment:
This isn't going to be virtual anymore. Is that fine?

Reply:
Thank you for pointing that out. I made a mistake in the correction method.

    alias OleClientSite.AddRef AddRef;

In dmd 2.087.0, A same module member can't be access this `AddRef`.

    override protected int AddRef() { return super.AddRef(); }

This code is fine. But I can't tell if this is a compiler spec change or a bug.

Reply:
The existing alias is trying to create a public symbol out of a protected symbol. That is not allowed. Previously the compiler did not do a proper access/visibility check.

I think this is what's kicking in: https://dlang.org/changelog/2.087.0.html#remove_visibility.